web-app/src/main/java/com/google/sps/data/Analysis.java

```diff
@@ -0,0 +1,56 @@
+package com.google.sps.data;
+
+import java.util.HashMap;
+import java.util.Map.Entry;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+
+public class Analysis {
+  private final int numReports;
+  private final List<String> frequentTypes;
+  private HashMap<String, Integer> crimeTypeCountMap;
+
+  private void countCrimeType(List<PoliceReport> reports) {
+    crimeTypeCountMap = new HashMap<String, Integer>();
+
+    for (PoliceReport report : reports) {
+      String crimeType = report.getCrimeType();
+      Integer count = crimeTypeCountMap.get(crimeType);
```

Comment:
Lowercase int?

Reply:
It needs to be Integer because the hashmap is of type <String, Integer> :(